modules/saml/lib/Auth/Source/SP.php

```diff
@@ -95,13 +102,20 @@ public function __construct($info, $config)
         $this->idp = $this->metadata->getString('idp', null);
         $this->discoURL = $this->metadata->getString('discoURL', null);
         $this->disable_scoping = $this->metadata->getBoolean('disable_scoping', false);
+        $this->proxyLogoutEnabled = $this->metadata->getBoolean('proxyLogoutEnable', false);
```

Comment:
A new metadata option appears but the docs are not updated to mention it.

Reply:
My bad. Thanks for pointing out.  
I just added the missing option documentation.